ObjectOrientedProgramming/Memory Ownership/Object Lifetime/task.md

```diff
@@ -0,0 +1,35 @@
+As we remember from the 'Classes and Objects' block, the constructor and destructor are mandatory elements of any class, and they are responsible for creating and destroying an object.
+
+In this module, we will look at different topics, including memory ownership, the lifetime of an object, and such useful tools as smart pointers.
+
+### Lifetime of the object
+
+Each reference and object has a lifetime during the execution time of the program, it depends on the time of creation and destruction of this object. The lifetime of an object begins when the program allocates the required amount of memory to the object and initializes it, including default initialization. The lifetime ends at the moment when the object is manually deleted (for example, when calling the destructor) when exiting the scope of a code block (when we meet the `}` closing curly bracket), or when the place given to the object is occupied by another object or is cleared.
+
+### Storage duration
```

Comment:
Storage duration was in fact covered in module 2. 
(Although, I probably never references this term directly --- perhaps we should change that).

So I think we can remove this info from this task.